yarn-project/circuits.js/package.json

```diff
@@ -29,17 +29,6 @@
     "remake-constants": "node --loader ts-node/esm src/scripts/constants.in.ts && prettier -w src/constants.gen.ts && cd ../../l1-contracts && ./.foundry/bin/forge fmt",
     "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests"
   },
-  "inherits": [
```

Comment:
removing this, otheriwse it breaks yarn-project-base when merging to master. See here https://aztecprotocol.slack.com/archives/C03P17YHVK8/p1707752827905929

I think the solution is to just add jest.config.ts to all the yarn projects instead of configuring from package.json.